src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+public class Book {
+    private long id;
+    private String title;
+    private int price;
+
+    private Book(long id, String title, int price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
It's not consistent to use both a constructor and setters for object initialization. Since you have setters for all fields, consider using a no-argument constructor and initializing fields through setters, or remove the setters and use only the constructor for initialization.